### ISSUE TITLE
failed to show error report when using Symbol

### ISSUE THREAD

**Issue by mohayonao** [power-assert-js/power-assert#23]
```js
describe("test", () => {
  const FOO = Symbol("FOO");
  const BAR = Symbol("BAR");
  const BAZ= Symbol("BAZ");

  let obj = {
    [FOO]: 100,
    [BAR]: 100,
    [BAZ]: 200,
  };

  it("works", () => {
    assert(obj[FOO] === obj[BAR]);
  });
  it("does not works", () => {
    // this test fails
    assert(obj[FOO] === obj[BAZ]);
  });
});
```

I expected:

```
1) test does not works:
  
  assert(obj[FOO] === obj[BAZ])
         |  ||    |   |  ||    
         |  ||    |   |  |Symbol(BAZ)
         |  ||    |   |  200   
         |  ||    |   Object{FOO:100,BAR:100,BAZ:200}
         |  ||    false        
         |  |Symbol(FOO)             
         |  100                
         Object{FOO:100,BAR:100,BAZ:200}
  
  [number] obj[BAZ]
  => 200
  [number] obj[FOO]
  => 100
```

but got:
```
  1) test does not works:
     TypeError: Cannot read property 'filter' of null
      at Array.forEach (native)
      at Context.<anonymous> (test/build.js:22:5)
```

**Comment by twada** [comment]
Hi @mohayonao, thanks for reporting and reproduction case.

Oh `Symbol`s ... okay I'll check and fix the bug.

**Comment by mohayonao** [comment]
I'm checking this issue in `babel-node` and `node`.

**Comment by twada** [comment]
Just released [stringifier v1.2.0 with ES6 Symbol support](https://github.com/twada/stringifier/releases/tag/v1.2.0)

For the quick-fix solution, re-installing power-assert will fix this problem.
`npm uninstall power-assert && npm install power-assert`

Of course, I'll release power-assert 0.11.1 or 1.0.0 soon.

**Comment by twada** [comment]
This issue is fixed by the [1.0.0 release](https://github.com/power-assert-js/power-assert/releases/tag/v1.0.0).
Feel free to reopen this if it is not fixed.
